tests/console/btrfsmaintenance.pm

```diff
@@ -56,7 +56,19 @@ sub run {
     assert_script_run('/usr/share/btrfsmaintenance/btrfs-balance.sh', timeout => 300);
     assert_script_run('/usr/share/btrfsmaintenance/btrfs-scrub.sh ',  timeout => 300);
     assert_script_run('systemctl restart btrfsmaintenance-refresh.service');
-    assert_script_run("systemctl is-enabled btrfsmaintenance-refresh");
+    # Check state of btrfsmaintenance-refresh units. Have to use (|| :) due to pipefail
+    assert_script_run('(systemctl is-enabled btrfsmaintenance-refresh.path || :) | grep enabled');
+    # Fixed in SP1:Update, but SP3 is based on an older snapshot currently
+    if (is_sle("<15-SP1") || is_sle("=15-SP3") || is_leap("=15.3")) {
+        assert_script_run('(systemctl is-enabled btrfsmaintenance-refresh.service || :) | grep enabled');
+        record_soft_failure('boo#1165780 - Preset is wrong and enables btrfsmaintenance-refresh.service instead of .path');
+    } elsif (is_sle("<=15-SP3") || is_leap("<=15.3")) {
+        # Preset is correct, btrfsmaintenance-refresh.service still has the [Install] section
+        assert_script_run('(systemctl is-enabled btrfsmaintenance-refresh.service || :) | grep disabled');
```

Comment:
Sorry, this parts is a bit ambiguous to me, shouldn't the branch cover 15-SP2/15.2 and older versions?

Reply:
Ok, the first condition will go away completely. Pls, ignore.